src/search_algo.hpp

```diff
@@ -1166,9 +1165,9 @@ iterateMatchesFullSerial(TLocalHolder & lH)
         bm._n_qId = it->qryId / seqan::qNumFrames(TGlobalHolder::blastProgram);
         bm._n_sId = it->subjId / seqan::sNumFrames(TGlobalHolder::blastProgram);
 
-        bm.sLength = seqan::sIsTranslated(TGlobalHolder::blastProgram)
-                        ? lH.gH.indexFile.seqs[it->subjId].size()
-                        : lH.gH.transSbjSeqs[it->subjId].size();
+        bm.qLength = //std::ranges::size(lH.gH.transQrySeqs[it->qryId]);
```

Comment:
What about the comment? Is this needed in the future?

Reply:
We should discuss this again. Like I said, I think basing the variable on the actual translated seq length would be better for the e-value calculation but for now the focus was on reproducing lambda2's results.